python-spec/src/somacore/experiment.py

```diff
@@ -47,8 +47,8 @@ def axis_query(
         self: _Self,
         measurement_name: str,
         *,
-        obs_query: query.AxisQuery = query.AxisQuery(),
-        var_query: query.AxisQuery = query.AxisQuery(),
+        obs_query: Optional[query.AxisQuery] = None,
+        var_query: Optional[query.AxisQuery] = None,
```

Comment:
docstring for this method indicates `See :class:`query.ExperimentAxisQuery` for details on usage`.  But in fact I think the problem will still persist in `somacore` code.  If caller of this methods specifies `None`, somacore will not convert that to AxisQuery().  Does the fix need to be pushed down into somacore perhaps? Either way, suggest adding a test in this repo to ensure correct behavior.